helpers/all.js

```diff
@@ -3,25 +3,19 @@
 const _ = require('lodash');
 
 /**
- * Yield block only if all arguments are valid
+ * Yield block only if all args are valid
  *
  * @example
  * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/all}}
  */
 const factory = () => {
-    return function() {
-        var args = [], opts, result;
+    return function(...args) {
 
-        // Translate arguments to array safely
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
-        }
-
-        // Take the last argument (content) out of testing array
-        opts = args.pop();
+        // Take the last arg out of args array
+        const opts = args.pop();
```

Comment:
@bc-williamkwon why we need to add const?

Reply:
@bc-zoharmuzafi `opts` was listed as a var before at line 13 but since that line was removed it needs to be instantiated.  Before it was reassigned and was thus a var, but as it is never reassigned it can be a const here.